Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.9-slim
+
+WORKDIR /service
+COPY . /service
+RUN pip install -r requirements.txt
+
+EXPOSE 5000
+
+# create new user and execute as that user
+RUN useradd --create-home appuser
+WORKDIR /home/appuser
+USER appuser
+
+ENTRYPOINT ["python"]
```

Comment:
just as a reminder: this needs to be updated to the service CLI.

Reply:
Yes, I wanted to check with you on the service CLI. Should we also define boiler plate for the CLI and config as well? Or is the config part of the shared libs?

Reply:
Eventually, it would be great to have that in the shared libs.